docs/overview.md

```diff
@@ -395,14 +422,14 @@ Detects usages of formatting functions without formatting arguments.
 **Before:**
 ```go
 fmt.Sprintf("whatever")
-fmt.Errorf("whereever")
+fmt.Errorf("wherever")
```

Comment:
Happened due to docs re-generation.
And I think there is a typo: should be `whatever`, not `wherever`. Right?